_includes/mctheme.html

```diff
@@ -13,33 +13,33 @@
 
 		<p>To use this theme, include the theme file <b><i>after</i></b> the complete Bootstrap CSS and the complete Fuel UX CSS. For more on setting up a page, please review <a href="getting-started.html">Getting Started</a> and modify the <a href="getting-started.html#template">quick-start templates</a> with the following:</p>
 {% highlight html %}
-<link href="{{site.bootstrap.css}}" rel="stylesheet">
-<link href="{{site.cdn.css}}" rel="stylesheet">
-<link href="{{site.mctheme.css}}" rel="stylesheet">
+<link href="//maxcdn.bootstrapcdn.com/bootstrap/3.3.4/css/bootstrap.min.css" rel="stylesheet">
+<link href="//www.fuelcdn.com/fuelux/3.x.x/css/fuelux.min.css" rel="stylesheet">
+<link href="//www.fuelcdn.com/fuelux-mctheme/1.x.x/css/fuelux-mctheme.min.css" rel="stylesheet">

```

Comment:
No, it's intentionally wrong. 

This page is not being updated regularly. MCtheme is still at 1.0.0 and Fuel UX is at 3.6.3. 

That's why I added the "Please refer to the repository for the latest release." line.